.github/workflows/create-sdk-update-pr.yml

```
@@ -43,19 +43,90 @@ jobs:
           git push origin --delete ${{ steps.branching.outputs.branch_name }} || true
           git checkout -b ${{ steps.branching.outputs.branch_name }}
 
+      - name: Bump iOS player SDK version
+        if: ${{ inputs.sdk_name == 'ios' }}
+        run: |
+          sed -i '' 's/s.dependency "BitmovinPlayer", ".*/s.dependency "BitmovinPlayer", "${{ inputs.version_number }}"/g' ios/RNBitmovinPlayer.podspec
+
+      - name: Bump Android player SDK version
+        if: ${{ inputs.sdk_name == 'android' }}
+        run: |
+          sed -i '' "s/com.bitmovin.player:player:.*/com.bitmovin.player:player:${{ inputs.version_number }}'/g" android/build.gradle
+          sed -i '' "s/com.bitmovin.player:player-media-session:.*/com.bitmovin.player:player-media-session:${{ inputs.version_number }}'/g" android/build.gradle
+
+      - name: Commit version bump
+        run: |
+          git add ios/RNBitmovinPlayer.podspec android/build.gradle
+          git commit --no-verify -m "chore(${{ inputs.sdk_name }}): update ${{ inputs.sdk_name }} player version to ${{ inputs.version_number }}"
+          git push origin ${{ steps.branching.outputs.branch_name }}
+
+      - name: Create PR
+        run: |
+          gh pr create \
+          --base "${{ github.ref }}" \
+          --title "Update ${{ inputs.sdk_name == 'ios' && 'iOS' || inputs.sdk_name }} player to ${{ inputs.version_number }}" \
+          --body "Automated ${{ inputs.sdk_name == 'ios' && 'iOS' || inputs.sdk_name }} player version update to ${{ inputs.version_number }}"
+
+  update-v0:
+    name: Update SDK version on v0
+    runs-on: macos-14
```

Why not using `macos-15` here as well? I'd switch to either that, or

```suggestion
    runs-on: macos-latest
```